src/CachePlugin.php

```diff
@@ -138,6 +137,40 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * Calculate the timestamp when this cache item should be dropped from the cache. The lowest value return after
+     * the calculation is $maxAge.
+     *
+     * @param int|null $maxAge
+     *
+     * @return int|null Unix system time. @see PSR6 for caching
```

Comment:
I do not think so because 
* it is a private method, 
* it is described in the docs
* it is not really true

``null`` means that the cache implementation store it for its default value. If you are using a memory (array) cache it will be dropped after the request. 

I think it is enough to reference the PSR-6